lib/diffit/timestamp.rb

```diff
@@ -0,0 +1,25 @@
+# encoding: utf-8
+
+module Diffit
+  # Provides a value converter to datetime
+  #
+  module Timestamp
```

Comment:
Yes, i think so. Actually at first it were a class, inherited from a `::Timestamp`, but its initializer proves to be overkill.

Well, because this is the only util function, I think we could just rename the module.